service/lib/agama/storage/configs/with_alias.rb

```diff
@@ -33,6 +33,11 @@ module WithAlias
         def alias?(value)
           self.alias == value
         end
+
+        # Ensures the config has a value for alias.
+        def ensure_alias
+          self.alias ||= "device#{rand(10**9)}"
+        end
```

Comment:
In yast-storage-ng we use SecureRandom, which feels more... well... secure. ;-)

https://github.com/yast/yast-storage-ng/blob/00c076d27ef8186398cd47cccde95588a90601e5/src/lib/y2storage/planned/device.rb#L61